### PR TITLE
Final (maybe?) copy changes and tweaks for launch

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change
- Updates copy
- Small UI fixes
- Changes the `countOverTime` metric which used to also encompass projections to `countOverTime` and `projectedCountOverTime` to remove the redundant header in the methodology page.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [ ] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/CoreStore/MetricsStore.ts [hunk 1]
```diff
@@ -74,10 +73,9 @@ export default class MetricsStore {
         [PATHWAYS_SECTIONS.countByRace]: this.libertyToPrisonPopulationByRace,
       },
       [PATHWAYS_PAGES.prison]: {
-        [PATHWAYS_SECTIONS.countOverTime]:
-          this.rootStore.currentTenantId === US_ID
-            ? this.projectedPrisonPopulationOverTime
-            : this.prisonPopulationOverTime,
+        [PATHWAYS_SECTIONS.projectedCountOverTime]: this
+          .projectedPrisonPopulationOverTime,
+        [PATHWAYS_SECTIONS.countOverTime]: this.prisonPopulationOverTime,
```
Comment (Catacola) [inline src/core/CoreStore/MetricsStore.ts:78]: Agreed. Much cleaner.

File: src/core/content/metric/default.ts [hunk 2]
```diff
@@ -32,20 +32,22 @@ const content: MetricCopy = {
...
   },
   prisonPopulationOverTime: {
-    title: "Prison population over time",
+    title: "Incarceration population over time",
     methodology:
-      "The chart describes the historical prison population over the selected “Time Period”. Each data point represents the total population for the selected group on the first day of that month. For example, hovering over the “November 2020” data point on the chart will show the total number of people described in the section above on November 1, 2020. ",
+      'The chart describes the historical incarceration population over the selected "Time Period". Each data point represents the total population for the selected group on the first day of that month. For example, hovering over the "November 2020" data point on the chart will show the total number of people described in the section above on November 1, 2020. ',
```
Comment (Catacola) [inline src/core/content/metric/default.ts:45]: Thanks for catching this. I made an issue to fix up the script to handle quotes better. Good for a bug bash. https://github.com/Recidiviz/pulse-dashboard/issues/1642

File: src/core/models/PrisonPopulationOverTimeMetric.ts [hunk 3]
```diff
@@ -114,6 +114,9 @@ export default class PrisonPopulationOverTimeMetric extends PathwaysMetric<Priso
     this.dataSeries.forEach((d: PrisonPopulationTimeSeriesRecord) => {
       data.push({
         Population: Math.round(d.count),
+        ...(d.avg90day && {
+          "3-month rolling average": Math.round(d.avg90day),
+        }),
```
Comment (Catacola) [inline src/core/models/PrisonPopulationOverTimeMetric.ts:119]: Is `avg90day` ever not set? According to the typedefs in `models/types.ts`, it's a required field.

File: src/core/content/metric/default.ts [hunk 4]
```diff
@@ -32,20 +32,22 @@ const content: MetricCopy = {
...
   },
   prisonPopulationPersonLevel: {
-    title: "List of people in prison",
+    title: "List of people in incarceration",
     methodology:
-      "The table includes a row for each person in prison as of the date specified in the chart title. ",
+      "The table includes a row for each person in incarceration as of the date specified in the chart title. ",
```
Comment (Catacola) [inline src/core/content/metric/default.ts:50]: Might be a note for elsewhere, but `in incarceration` feels weird. Why not `incarcerated`? @sychang

**Review by Catacola — APPROVED** [review]

File: src/core/models/PrisonPopulationOverTimeMetric.ts (hunk 3 above)
Comment (jovergaag) [reply src/core/models/PrisonPopulationOverTimeMetric.ts:119]: It is not set on the time series metrics. Only on the count by month metrics.

**Pull request merged**